submissions/sprint_b/app.py

```
@@ -0,0 +1,24 @@
+from sprint_b import app
+from flask import jsonify
+from helpers import EpithetGenerator
+
+
+@app.route('/')
+def generate_epithets():
+    single_epithet = EpithetGenerator.display_single_epithet()
+    return jsonify(single_epithet)
+
+
+@app.route('/vocabulary')
+def vocabulary():
+    vocab_dataset = EpithetGenerator.display_vocab_dataset()
+    return jsonify(vocab_dataset)
+
+
+@app.route('/epithets/<quantity>')
+def quantity(quantity):
+    epithets = EpithetGenerator.get_quantity_of_epithets(int(quantity))
+    return jsonify(epithets)
+
+
+app.run()
```

app.run  should only run if __name__ == '__main__' otherwise, it throws the following warning

"Warning: Silently ignoring app.run() because the application is run from the flask command line executable.  Consider putting app.run() behind an if __name__ == "__main__" guard to silence this warning."